tests/ga/test_extension.py

```diff
@@ -367,6 +367,8 @@ def tearDownClass(cls):
 @patch('time.sleep', side_effect=lambda _: mock_sleep(0.001))
 @patch("azurelinuxagent.common.protocol.wire.CryptUtil")
 @patch("azurelinuxagent.common.utils.restutil.http_get")
+@patch('azurelinuxagent.common.protocol.extensions_config_retriever.ExtensionsConfigRetriever._get_saved_incarnation', MagicMock(return_value=0))
+@patch('azurelinuxagent.common.protocol.extensions_config_retriever.ExtensionsConfigRetriever._get_saved_sequence_number', MagicMock(return_value=1))
```

Comment:
will _get_saved_sequence_number go away once we add the timestamp?
if will still need this mock, let's sync offline

Reply:
These mocks were removed